docs/components/AuthNavButton.tsx

```diff
@@ -32,13 +32,19 @@ export function AuthNavButton(props: any) {
           </button>
         </Menu.Item>,
       ]}>
-      <Image
-        className="size-5 rounded-md"
-        src={session.data.user!.image!}
-        alt={session.data.user!.name!}
-        width={50}
-        height={50}
-      />
+      {session.data.user.image ? (
+        <Image
+          className="size-6 rounded-full"
+          src={session.data.user.image}
+          alt={session.data.user.name}
+          width={50}
+          height={50}
+        />
+      ) : (
+        <div className="flex size-6 items-center justify-center rounded-full bg-indigo-400 text-xs font-semibold dark:bg-indigo-600">
+          {session.data.user.name.substring(0, 2).toUpperCase()}
+        </div>
```

Comment:
It's possible that the user does not have a name set (like magic link sign in)
<img width="189" alt="image" src="https://github.com/user-attachments/assets/21d569b6-7736-49b4-896f-0577dc920fc8" />

Maybe a person icon instead? Although I do like the initials